packages/web/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 0.2.0
+## 0.2.1 2022-6-27
+
+- Fixed errors in common dependency. 
```

Comment:
Please follow the format:
```
### Fixed
- Errors in common dependency.
```
Also, it would be better to be more exact about what error that was.

Reply:
I have clarified.